frontend/packages/rhoas-plugin/src/utils/conditionHandler.ts

```diff
@@ -0,0 +1,26 @@
+import { StatusCondition } from './rhoas-types';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+export const getCondition = (request: K8sResourceKind, name: string) => {
+  if (request && request.status && request.status.conditions) {
+    for (const condition of request.status.conditions) {
+      if (condition.type === name) {
+        return condition as StatusCondition;
+      }
+    }
```

Comment:
I think we can make use of filter func here.

Reply:
yes. This is typical find method use case